test/Microsoft.IdentityModel.Tokens.Tests/TokenValidationParametersTests.cs

```diff
@@ -185,6 +200,31 @@ public void GetSets()
             Assert.Null(validationParameters.SignatureValidator);
         }
 
+        [Fact]
```

Comment:
>Fact [](http://example.com/codeflow?start=9&length=4)

Just to check my understanding, now that this is in place, it will be leveraged in SAL and that's where we should have tests to ensure properties get added to the instanced copy vs the root/normal property bag, correct?

Reply:
Yes